prow/git/git.go

```diff
@@ -86,7 +93,8 @@ func NewClient() (*Client, error) {
 		logger:    logrus.WithField("client", "git"),
 		dir:       t,
 		git:       g,
-		base:      fmt.Sprintf("https://%s", github),
+		base:      fmt.Sprintf("https://%s", host),
+		host:      host,
```

Comment:
How does this interact with SetRemote?

Reply:
Ok, so the problem has been there even before the change: `SetRemote` can update the base, but in `Clone, it is always using github constant.

I can update `SetRemote`, and parse the remote parameter, then update `host` field. However, @stevekuznetsov might want to see less changes here. Thoughts?

Reply:
Add a TODO round SetRemote at least then?

Reply:
Yeah, added a TODO comment on `SetRemote`.